tests/aws/test_integration.py

```diff
@@ -102,7 +102,7 @@ def test_firehose_s3(self, firehose_create_delivery_stream, s3_create_bucket, aw
         assert TEST_TAGS == tags["Tags"]
 
         # create target S3 bucket
-        aws_client.s3.create_bucket(Bucket=bucket_name)
+        s3_create_bucket(Bucket=bucket_name)
```

Comment:
Using this fixture ensures the the appropriate `LocationConstraint` is added automatically.